.github/workflows/release-chart.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      # Workaround DCO requirement for gh-pages branch
+      - run: git config alias.commit "commit -s"
       - name: Publish Helm charts
         uses: stefanprodan/helm-gh-pages@v1.1.0
```

Comment:
Should we update to v1.3.0 as well?

Reply:
Looking at the changes, I don't see anything that we need or would use. I could make that a separate change if there's something I'm missing.
https://github.com/stefanprodan/helm-gh-pages/releases